api-schema/src/main/java/org/stellar/anchor/api/sep/sep6/Transaction.java

```diff
@@ -0,0 +1,82 @@
+package org.stellar.anchor.api.sep.sep6;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Transaction {
```

Comment:
💡 rename to `Sep6Transaction`? In a few cases, we need to work with the `Sep24Transaction`, `Sep31Transaction` and `Sep6Transaction` classes.